crdstore/config/config.go

```diff
@@ -0,0 +1,16 @@
+package config
+
+import (
+	"time"
+
+	"github.com/allegro/akubra/types"
+)
+
+// CredentialsStore - config for credential store
+type CredentialsStore struct {
+	Endpoint types.YAMLUrl `yaml:"Endpoint"`
+	TTL      time.Duration `yaml:"TTL"`
```

Comment:
Is `time.Duration` serialized in human readable string or nanoseconds?

Reply:
I've changed it to metrics.Interval so it's human reable now